src/web/components/ParticipantRequests/ApprovedParticipantItem.tsx

```diff
@@ -0,0 +1,43 @@
+import { ParticipantRequestDTO } from '../../../api/routers/participantsRouter';
+
+import './ParticipantRequestItem.scss';
+
+type ApprovedParticipantProps = {
+  participant: ParticipantRequestDTO;
+};
+
+export function ApprovedParticipantItem({ participant }: ApprovedParticipantProps) {
+  function getParticipantTypes(
```

Comment:
I'm guessing this will end up using the types from admin rather than what they requested?

Reply:
Once approved, yes.  I'll add a note for when I return to this.